### PR TITLE
Maximize/ minimize option

### PR DESCRIPTION
**Pull request by nunos7**
Hi Sebastian,
I needed to add maximize option to the widgets.
I saw the existing issue (https://github.com/sdorra/angular-dashboard-framework/issues/32) and developed the according code.
I hope the code proposed meets your expectations :)

NS

### CODE REVIEW

**Comment by sdorra** [comment]
The changes are hard to read, because there a lot whitespace changes. It is possible that you recreate the pull request with two spaces (no tabs) instead of 4?

**Comment by sdorra** [comment]
I think the maximize option should be optional and disable by default. We could add another directive attribute to activate the maximize option. What do you think?

**Comment by nunos7** [comment]
Hi Sebastian,
I am taking the first steps in angularJS and github :(, if you think the best thing is to have a directive, for me it's ok :).

I cleaned the code and hope that is ok now.

Thanks, and good work.
NS

File: sample/scripts/sample.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ angular.module('sample', [
   'sample.widgets.version', 'LocalStorageModule',
   'structures', 'sample-01', 'sample-02', 'sample-03',
   'sample-04', 'ngRoute'
-])
+]).value('adfVersion', '0.6.1')
```
Comment (sdorra) [inline sample/scripts/sample.js]: Why do you add the version to the sample module?

**Comment by sdorra** [comment]
I'm very sorry, but can you squash the commits to a single commit?

http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit

File: sample/scripts/sample.js (hunk 1 above)
Comment (nunos7) [inline sample/scripts/sample.js]: Sorry my mistake please remove!

**Comment by nunos7** [comment]
Hi Sebastian, 
I will try to squash the commits to a single commint...

Sorry once again,
NS

**Comment by nunos7** [comment]
Hi Sebastian,
As I said I am newbie in "git", and I am not able to put the "pull request" as you intend. If you check the code with the changes (https://github.com/sdorra/angular-dashboard-framework/pull/70/files?diff=unified) I think it will not take more than 20 minutes to put these changes in your branch.

I apologize for the inconvenience :(
NS

**Pull request closed without merging**